hedgehog/src/Hedgehog/Internal/Property.hs

```diff
@@ -221,14 +237,23 @@ newtype PropertyName =
       unPropertyName :: String
     } deriving (Eq, Ord, Show, IsString, Semigroup, Lift)
 
+-- | The acceptable occurrence of false positives
+--
+--   Example, @Confidence 10^9@ would mean that you'd accept a false positive
+--   for 1 in 10^9 tests.
+newtype Confidence =
+  Confidence {
+    unConfidence :: Int64
+  } deriving (Eq, Ord, Show, Num, Lift)
+
 -- | Configuration for a property test.
 --
 data PropertyConfig =
   PropertyConfig {
-      propertyTestLimit :: !TestLimit
```

Comment:
removing this field broke the API between 1.0.1 and 1.0.2 :disappointed:

Reply:
Where's MIMA when you need it 😢

Reply:
Was it removed by accident? Also—it's time ti start adding some more automated tests.

Reply:
I think everything in `Hedgehog.Internal.Property` is considered internal.

Reply:
Was it removed for a reason though? (We *do* try not to break other peoples code if possible.)